### PR TITLE
Cyd/sitecleanup2

### PR DESCRIPTION
**Pull request by cydoval**
Changes to layouts and styles across the site

### CODE REVIEW

File: Harvest.Web/ClientApp/src/Quotes/ProjectDetail.tsx [hunk 1]
```diff
@@ -120,46 +120,20 @@ export const ProjectDetail = (props: Props) => {
...
-                </InputGroupAddon>
-                <Input
-                  type="number"
-                  id="rate"
-                  value={props.quote.acreageRate}
-                  disabled
-                />
-              </InputGroup>
+            <Col md="2">
+              <Label for="rate">Rate</Label> <br />${props.quote.acreageRate}
```
Comment (srkirkland) [inline Harvest.Web/ClientApp/src/Quotes/ProjectDetail.tsx:124]: let's throw a `formatCurrency()` in here so it looks nice, like we do with total down below

**Review by srkirkland — APPROVED** [review]

File: Harvest.Web/ClientApp/src/Quotes/ProjectDetail.tsx (hunk 1 above)
Comment (cydoval) [reply Harvest.Web/ClientApp/src/Quotes/ProjectDetail.tsx:124]: So like this? 
<Label for="rate">Rate</Label> <br />${formatCurrency(props.quote.acreageRate)}

Comment (cydoval) [reply Harvest.Web/ClientApp/src/Quotes/ProjectDetail.tsx:124]: `<Label for="rate">Rate</Label> <br />${formatCurrency(props.quote.acreageRate)}`

Comment (srkirkland) [reply Harvest.Web/ClientApp/src/Quotes/ProjectDetail.tsx:124]: yes perfect

**Pull request merged**